### PR TITLE
DOP-1418 - Promo codes

### PR DESCRIPTION
**Pull request by andresmoschini**
In this PR we are adding a listener for the `getPromoCodes` message, asking the backend for the data, and returning the result to the editor's extensions following the convention defined in https://github.com/FromDoppler/unlayer-editor/pull/927

- [x] I need to add the tests before merging

### CODE REVIEW

File: src/abstractions/editor-extensions-bridge/index.ts
Comment (andresmoschini) [inline src/abstractions/editor-extensions-bridge/index.ts:1]: Two things about it. 

1. I used abstractions and implementation modules in the wrong way.
2. I will split EditorExtensionsBridge into two parts: one to deal with events and messages, and the other one implements the listeners.

File: src/abstractions/doppler-legacy-client/index.ts [hunk 1]
```diff
@@ -10,6 +10,18 @@ export type UploadImageError =
...
+export type PromoCodeItem = {
+  code: string;
+  type: "percent" | "money";
+  value: number;
+  startDate: Date | undefined;
+  endDate: Date | undefined;
+  useLimit: number | undefined;
+  minPaymentAmount: number;
+  promotionName: string;
+  isActive: boolean;
```
Comment (leoslopez) [inline src/abstractions/doppler-legacy-client/index.ts:22]: These fields are not being considered from [unlayer-extension side](https://github.com/FromDoppler/unlayer-editor/blob/main/src/customJs/properties/promo_codes/types.ts#L10), but I think that they are not necessary, right?

Comment (andresmoschini) [reply src/abstractions/doppler-legacy-client/index.ts:22]: You are right, at the moment they are not necessary but I think that they could be useful in the future.

**Review by leoslopez — APPROVED** [review]
💪

**Pull request merged**